### PR TITLE
add manage/insertword function

### PR DESCRIPTION
**Pull request by Hang91**

### CODE REVIEW

File: dao.go [hunk 1]
```diff
@@ -45,3 +45,13 @@ func (u *UsersDAO) SearchWord(spell string) (bool, Word) {
...
 }
+
+// Insert word
+func (u *UsersDAO) InsertWord(word Word) (bool, error) {
+	err := db.C(COLLECTION_WORD).Insert(&word)
+	if err != nil {
+		log.Println(err)
+		return false, err
+	}
+	return true, nil
```
Comment (yingyingzhuzhu) [inline dao.go]: return false, err

since in this case, err == nil

**Review by yingyingzhuzhu — COMMENTED** [review]
in line 56, return false, err

since in this case, err == nil

File: main.go [hunk 2]
```diff
@@ -3,6 +3,7 @@ package main
 import (
 	"log"
 	"net/http"
+	"time"
```
Comment (yingyingzhuzhu) [inline main.go:6]: where is "time" from?

Comment (Hang91) [reply main.go:6]: I don't know.

File: word.go [hunk 3]
```diff
@@ -0,0 +1,17 @@
+package main
+
+import (
+	"strings"
+)
+
+// split string
+// n:鱼;vi:捕鱼;vt:捕鱼
```
Comment (yingyingzhuzhu) [inline word.go:8]: Do we really need this function?

**Review by yingyingzhuzhu — APPROVED** [review]
Looks good.

**Pull request merged**